backend/app/users.py

```diff
@@ -102,18 +116,30 @@ async def get_user_manager(
 
 
 bearer_transport = BearerTransport(tokenUrl="api/auth/jwt/login")
+cookie_transport = CookieTransport(
+    cookie_name="dev-kronikarz",
+    cookie_max_age=36000,
+    cookie_secure=False,
+    cookie_samesite="strict",
+)
 
 
 def get_jwt_strategy() -> JWTStrategy:
     return JWTStrategy(secret=SECRET, lifetime_seconds=3600)
 
 
-auth_backend = AuthenticationBackend(
-    name="jwt",
+token_backend = AuthenticationBackend(
+    name="token",
```

Comment:
Why the name changed?

Reply:
We now have a cookie auth backend, so you don't lose auth when refreshing the docs